ci/testkomodo.sh

```diff
@@ -0,0 +1,13 @@
+
+copy_test_files () {
+    cp -r $CI_SOURCE_ROOT/tests $CI_TEST_ROOT/tests
+    cp $CI_SOURCE_ROOT/pyproject.toml $CI_TEST_ROOT
+}
+
+install_test_dependencies () {
+    pip install -e .
```

Comment:
Is this intended to work with the hook-setting thingies?
It probably needs this, ref our readme:

If you intend to run completor as a plugin to ert:

git clone https://github.com/equinor/completor.git
cd completor
python -m venv venv
source venv/bin/activate
pip install -e ".[ert]"

Reply:
This depends on how the tests are setup I guess.
I ran this locally, but have ert in my venv already, so maybe this is needed?

Reply:
I changed it to explicitly install everything it needs to run tests. Is this ok or should I change it back?